src/analysis/special_functions/trigonometric.lean

```diff
@@ -1141,13 +1141,25 @@ sin_add_pi_div_two x ▸ sin_pos_of_mem_Ioo ⟨by linarith [hx.1], by linarith [
 lemma cos_nonneg_of_mem_Icc {x : ℝ} (hx : x ∈ Icc (-(π / 2)) (π / 2)) : 0 ≤ cos x :=
 sin_add_pi_div_two x ▸ sin_nonneg_of_mem_Icc ⟨by linarith [hx.1], by linarith [hx.2]⟩
 
+lemma cos_nonneg_of_nonneg_of_le_of_le {x : ℝ} (hl : -(π / 2) ≤ x) (hu : x ≤ π / 2) :
```

Comment:
This lemma name seems to have too many `nonneg`s in it

Reply:
Should it be `cos_nonneg_of_le_of_le`?

Reply:
Or, wait... there should be a `pi_div_two` in there...